thirdparty/github.go

```diff
@@ -1556,6 +1556,27 @@ func userHasWritePermission(ctx context.Context, token, owner, repo, username st
 	return utility.StringSliceContains(githubWritePermissions, permissionLevel.GetPermission()), nil
 }
 
+func ValidateGitHubPermission(permission string) error {
+	if !utility.StringSliceContains(AllGitHubPermissions, permission) {
+		return errors.Errorf("invalid GitHub permission '%s'", permission)
+	}
+	return nil
+}
+
+func MostRestrictiveGitHubPermission(perm1, perm2 string) string {
```

Comment:
small nit: can you please add function comments for these two since they are exported functions?